docs/tutorial/mlflow-kubeflow.rst

```diff
@@ -0,0 +1,186 @@
+Getting Started with Charmed MLflow and Kubeflow
+================================================
+
++------------+---------+
+| Component  | Version |
++============+=========+
+| MLflow     | 2       |
++------------+---------+
+| Kubeflow   | 1.7     |
++------------+---------+
+
+Welcome to this tutorial on getting started with Charmed MLflow alongside Charmed Kubeflow. If you would like to deploy Kubeflow by itself, see our tutorial on `Charmed Kubeflow <https://charmed-kubeflow.io/docs/get-started-with-charmed-kubeflow>`_.
+
+Prerequisites
+-------------
+
+This tutorial assumes you will be deploying Kubeflow and MLflow on a public cloud VM with the following specs:
+
+- Runs Ubuntu 20.04 (focal) or later.
+- Has at least 4 cores, 32GB RAM and 100GB of disk space available.
+- Is connected to the internet for downloading the required snaps and charms.
+
+We’ll also assume that you have a laptop that meets the following conditions:
+
+- Has an SSH tunnel open to the VM with port forwarding and a SOCKS proxy. To see how to set this up, see `How to setup SSH VM Access <https://charmed-kubeflow.io/docs/how-tosetup-ssh-vm-access-with-port-forwarding>`_.
+- Runs Ubuntu 20.04 (focal) or later.
+- Has a web browser installed e.g. Chrome / Firefox / Edge.
+
+In the remainder of this tutorial, unless otherwise stated, it is assumed you will be running all command line operations on the VM, through the open SSH tunnel. It’s also assumed you’ll be using the web browser on your local machine to access the Kubeflow and MLflow dashboards.
+
+.. note:: With a few tweaks to the instructions, it is possible to do this entire tutorial on a laptop, but running it on a VM will give you a better feel for deploying Kubeflow and MLflow in production. So, we recommend using the VM, even if your laptop is powerful enough to deploy Kubeflow.
+
+Deploy MLflow
+-------------
+
+Follow the steps in this tutorial to deploy MLflow on your VM: :doc:`mlflow`. Before moving on with this tutorial, confirm that you can now access the MLflow UI on http://localhost:31380.
+
+Deploy Kubeflow bundle
+----------------------
+
+Let's deploy Charmed Kubeflow alongside MLflow. Run the following command to initiate the deployment:
+
+.. code-block:: bash
+
+   juju deploy kubeflow --trust  --channel=1.7/stable
+
+Be patient. Kubeflow is large and will take a few minutes to deploy.
```

Comment:
That's the understatement of the year, a few is 5' not 45'😂

Reply:
:laughing: Ssssssh